src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -11,6 +11,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Threading.Tasks.Dataflow;
+using Microsoft.VisualStudio.ProjectSystem.UpToDate;
```

Comment:
Yes, mistake that they were inconsistent. We seem to be inconsistent in our project system projects--some files in folders have sub namespaces, some don't. I'll just put them in sub-namespaces.